packages/components/src/templates/next/components/internal/SearchableTable/constants.ts

```diff
@@ -0,0 +1,6 @@
+// This is the maximum number of columns that the table can have
+// 10 was decided because at 1240px, each column will only have 124px which is
+// the minimum width for a column to be readable
+export const MAX_NUMBER_OF_COLUMNS = 10
```

Comment:
in mobile will still look weird right haha. Might as well set min-width on each cell. Can always enhance it (and our other tables?) with something like https://www.npmjs.com/package/react-table-column-resizer (or write our own/vendorize) to allow users to resize columns.

Reply:
If we set min-width (e.g., 124px) on the cols, will a table with 10 cols overflow and hori scroll on w=1025px (desktop)?

Reply:
I think with the borders likely will overflow, so maybe we can use a smaller number such that the min-width of all columns + borders is <= 1024px?

Reply:
Hmm I don't think overflow is necessarily bad when you have 10 (!!) columns, because at that point your data table is so big, that it's more important for us to guarantee that the column texts are readable than trying to avoid overflow. I'd rather the table be scrollable than we try to squeeze 10 columns into {1024px - margins} which might break words at awkward places.

I guess no issue in setting a min-width on columns, but should we be setting a max-width on the entire table instead? So we let the columns resize depending on content but prevent the scroll being ridiculously long, like 2* of the screen width